docs/examples/wpi-many/wpi-many-custom-checker-example.sh

```diff
@@ -71,6 +96,6 @@ bash wpi-many.sh -o "${workingdir}/${checkername}-${repolistbase}" \
      -t ${timeout} \
      -- \
      --checker "${checker}" \
-     --quals "${qual_classpath}" \
-     --lib "${checker_classpath}" \
-     --stubs "${custom_stubs}"
+     "${quals_arg}" \
```

Comment:
I wouldn't think these variables should be quoted, because they represent multiple tokens.  If this works as is, then fine.

Reply:
My understanding has always been that variables that might be empty or contain whitespace should be quoted. This version also passes shellcheck as-is, though I didn't actually run the script (I don't have the environment to do so anymore).

Reply:
If the variable is empty, your goal is not to pass a single empty token to the program; rather, your goal is to pass no tokens to the program.

Shellcheck assumes you want to pass a single token; a warning suppression is necessary in cases like this.

Reply:
Thanks for the explanation. I looked through https://github.com/koalaman/shellcheck/wiki/SC2086 and found an alternative that avoids the shellcheck warning but also has the behavior we're looking for, I think, using alternate expansion (the discussion of this approach is near the bottom of that page).